lib/render/leaderboard.php

```diff
@@ -226,9 +226,9 @@ function RenderScoreLeaderboardComponent(string $user, bool $friendsOnly, int $n
                     echo GetUserAndTooltipDiv($userData[0]['User']);
                     echo "</td>";
                     if ($j == 0) {
-                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['Points'] . "</a>";
+                        echo "<td><a href='/historyexamine.php?d=$dateUnix&u=" . $userData[0]['User'] . "'>" . $userData[0]['HardcorePoints'] . "</a>";
                     } else {
-                        echo "<td>" . $userData[0]['Points'];
+                        echo "<td>" . $userData[0]['HArdcorePoints'];
```

Comment:
`HardcorePoints`

Reply:
I do this mistake too often...thanks again

Reply:
No worries! common typo for me as well. just like like "acheivements" :D not sure i can say i like typing that word...